apps/remix-ide/src/app/tabs/debugger/debuggerUI.js

```diff
@@ -186,12 +227,18 @@ class DebuggerUI {
     this.stepManagerView = yo`<div class="px-2"></div>`
 
     var view = yo`
-      <div>
+      <div>        
         <div class="px-2">
+          <div class="mt-3">
+            <p class="mt-2 ${css.debuggerLabel}">Debugger Configuration</p>
+              <div class="mt-2 ${css.debuggerConfig} custom-control custom-checkbox">
+              <input class="custom-control-input" id="debugGeneratedSourcesInput" onchange=${(event) => { this.opt.debugWithGeneratedSources = event.target.checked }} type="checkbox" title="Debug with generated sources">
+              <label data-id="debugGeneratedSourcesLabel" class="form-check-label custom-control-label" for="debugGeneratedSourcesInput">Debug Generated sources if available (from Solidity 0.7.2)</label>
```

Comment:
Minor correction: `Debug Generated sources if available (from Solidity 0.7.2)` => `Debug generated sources if available (from Solidity v0.7.2)`

Also can we check the compiler version from compilation data and show checkbox only if Solidity version is greater than v0.7.1 (That can be done in separate PR too)

Reply:
> Also can we check the compiler version from compilation data and show checkbox only if Solidity version is greater than v0.7.1 (That can be done in separate PR too)

That might be a bit complicated... cause we are also using sourcify for retrieving the compiler version.
i.e when the user input a transaction hash and click debug, we fetch the metadata.json and this is only at this point of time that we know the compiler version...